packages/tsconfig/package.json

```diff
@@ -2,12 +2,7 @@
   "name": "tsconfig",
   "version": "0.0.0",
   "private": true,
-  "files": [
-    "base.json",
-    "nextjs.json",
-    "react-library.json"
-  ],
-  "scripts": {
```

Comment:
can we keep the update script please? This way we can easily update packages througout the monorepo